atip/simulator.py

```diff
@@ -74,15 +74,14 @@ def __init__(self, at_lattice, callback=None, emit_calc=True):
         self._at_lat = at_lattice
         self._rp = numpy.ones(len(at_lattice) + 1, dtype=bool)
         self._emit_calc = emit_calc
+        self._at_lat.radiation_on()
         # Initial phys data calculation.
-        if self._emit_calc:
-            self._at_lat.radiation_on()
-            self._emitdata = self._at_lat.ohmi_envelope(self._rp)
-        self._at_lat.radiation_off()
-        self._lindata = self._at_lat.linopt(
-            refpts=self._rp, get_chrom=True, coupled=False
+        twiss0, self._beamdata, self._twiss = self._at_lat.linopt6(
```

Comment:
I believe this duplicates the code in `_recalculate_phys_data()`.

Could we extract this into a separate function?